system_baseline/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.0"
```

Comment:
Could there be a separate version file that __init__.py requires?

Reply:
ya we could do that. Would you prefer a version.py that `__init__.py` pulls in, and then the view reads from that, or if `version.py` was pulled in directly by the view?

Reply:
I could live with either. I just like knowing exactly where to see the version.